test/everypolitician_test.rb

```diff
@@ -195,4 +195,11 @@ def test_expose_the_statement_count_of_a_legislature
     end
   end
 
+  def test_lastmod_is_a_time
+    VCR.use_cassette('countries_json') do
+      australian_senate = Everypolitician.country(slug: 'Australia').legislature(slug: 'Senate')
+      assert_equal Time, australian_senate.lastmod.class
```

Comment:
I don't think test is sufficient — this would pass if the implementation were simply `Time.now`